### DISCUSSION TITLE
When I enter code no help appears

### DISCUSSION THREAD

**Discussion by Starlight7749** [SilentVoid13/Templater#1109] [Help]
I have seen videos of when i start typing <% options for additional info then appear, however it is not showing anything when i do it.

Am I missing a setting somewhere?

**Comment by Zachatoo** [comment]
1. It only works on desktop.
2. You have to type `tp.` within a `<% ... %>` block. It's the `tp.` that triggers the autocompletion.

**Reply by Starlight7749** [reply]
Thank you
